R/example.R

```diff
@@ -121,8 +124,11 @@ print_example_program <-
 #' f2 <- write_stan_tempfile(lines)
 #' identical(readLines(f), readLines(f2))
 #'
-write_stan_tempfile <- function(code) {
-  tmp <- tempfile(fileext = ".stan")
+write_stan_tempfile <- function(code, dir = tempdir()) {
```

Comment:
Just wanted to chime in to say thanks for this! I kept forgetting to do a feature request issue for this.